### PR TITLE
Move things

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/um/dbghelp.rs [hunk 1]
```diff
@@ -385,3 +385,121 @@ STRUCT!{struct IMAGEHLP_LINEW64 {
     Address: DWORD64,
 }}
 pub type PIMAGEHLP_LINEW64 = *mut IMAGEHLP_LINEW64;
+
+extern "system" {
+    pub fn EnumDirTree(
+        hProcess: HANDLE, RootPath: PCSTR, InputPathName: PCSTR, OutputPathBuffer: PSTR,
```
Comment (Susurrus) [inline src/um/dbghelp.rs]: All function arguments should be on a line by themselves. See the style guidelines.

**Comment by Susurrus** [comment]
This needs a rebase as #468 already took care of `netapi32` and it was already merged. Definitely a lot of stuff to keep track of here!

**Comment by GuillaumeGomez** [comment]
And updated.

**Comment by Susurrus** [comment]
You have some test failures with this.

**Comment by GuillaumeGomez** [comment]
Let's hope I didn't forget anything this time. T_T

**Comment by retep998** [comment]
Could you define the symbols in the order specified in the header? Thanks.

**Comment by GuillaumeGomez** [comment]
Done.

**Pull request merged**